Yosemite/YosemiteTests/Stores/ReceiptStoreTests.swift

```diff
@@ -515,6 +515,62 @@ final class ReceiptStoreTests: XCTestCase {
         let expectedError = expectedResult.failure
         XCTAssertNotNil(expectedError)
     }
+
+    func test_sendReceipt_when_order_updates_and_action_succeeds() throws {
```

Comment:
nit: can follow the Given/When/Then pattern in the new test cases (this and the next)
```suggestion
    func test_sendReceipt_when_order_updates_and_action_succeeds() throws {
        // Given
```